metricbeat/module/nats/_meta/docs.asciidoc

```diff
@@ -1,10 +1,16 @@
-This is the nats module.
-
-This is the Nats Module. The Nats module uses http://localhost:8222 APIs to collect metrics.
+This is the Nats module. The Nats module uses http://localhost:8222 APIs to collect metrics.
```

Comment:
Should we also link here to the Nats (monitoring docs) for some background?

Reply:
on its way